de.metas.shipper.gateway.derkurier/src/main/java/de/metas/shipper/gateway/derkurier/DerKurierDeliveryData.java

```diff
@@ -43,8 +45,12 @@ public static DerKurierDeliveryData ofDeliveryPosition(@NonNull final DeliveryPo
 	String customerNumber;
 
 	String parcelNumber;
-	
+
 	String customerCode;
-	
+
 	String collectorCode;
+
```

Comment:
* please prefer the new java time API to timestamp (LocalDateTime etc)
* we need so specify times without date. E.g. whenever something is delivered, it needs to be between 9:00 and 17:00
  * suggest to take a look at `LocalTime`

Reply:
Done.
Note that I had to also refactor some tests and I added some code to the TimeUtil.